package.json

```diff
@@ -17,6 +17,7 @@
     "express": "^4.16.4",
     "mongoose": "^5.10.0",
     "node-cron": "^3.0.0",
```

Comment:
I recommend incrementing the version.
Consider this a main-line update (eg. from 2.1.1 -> 2.2.0)

Reply:
Agreed